src/odemis/driver/pigcs.py

```diff
@@ -2419,7 +2419,7 @@ def startReferencing(self, axis):
             self.ReferenceToSwitch(axis)
         elif self._hasLimitSwitches[axis]:
             raise NotImplementedError("Don't know how to reference to limit yet")
-            self.ReferenceToLimit(axis)
+            self.ReferenceToLimit(axis)  #TODO code unreachable??
```

Comment:
Yes. Sure. It's just code in preparation to the day we'll implement the reference support.